### DISCUSSION TITLE
remove  the `Global values` at every css page

### DISCUSSION THREAD

**Discussion by zhangenming** [mdn/mdn-community#719] [Content] [answered]
Since these properties are the same on every page
Maybe we should remove it
just prompt it on a special page
![image](https://github.com/user-attachments/assets/684aa39a-35a9-475d-afbb-30211f8831ca)

**Comment by dipikabh** [comment]
Hi @zhangenming, we appreciate your interest in improving MDN!

While it's true that global values are universally applicable to all CSS properties, we have deliberately chosen to list them explicitly for each property. This is because we want every CSS property page to be a complete and self-contained reference in terms of all the values valid for the property. The "Syntax" section, therefore, lists all of them alongside property-specific values.

As an improvement, we could consider adding links to these keyword values (for example, links to [`initial`](https://developer.mozilla.org/en-US/docs/Web/CSS/initial), [`inherit`](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit), and so on) in the "Values" section to provide quick access to their detailed descriptions.

Let me know if you have any further thoughts.

**Comment by zhangenming** [comment]
I think the approach of `global attr` at hrml page is more better

maybe we need a page`https://developer.mozilla.org/en-US/docs/Web/css/Global_values`

![image](https://github.com/user-attachments/assets/be9193f0-a97d-44b2-8aa8-813575891181)

**Reply by dipikabh** [reply]
True, adding a similar statement to CSS property pages would be a good improvement. Your suggestion aligns with the idea I'd shared in [this discussion](https://github.com/orgs/mdn/discussions/462#discussioncomment-7721424).

We might not need a new page, we already have a few pages that list these CSS-wide keyword values: [css data types](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Types#css-wide_keywords), [css values and units](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#css-wide_values), [learn area](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance#controlling_inheritance). Among these, IMO, it would be best to link to the section on [css values and units](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#css-wide_values) page. So my proposal would be to add the following to the "Values" section in the CSS property pages:

> "Valid values also include the [CSS-wide keywords](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#css-wide_values)."

Let me bring this up in our next team meeting to see what everyone thinks about this approach. If there's consensus, I can open an issue to track the work to update the property pages.

**Comment by pransh15** [comment, marked as answer]
Hi @zhangenming 👋 

Thanks again for starting this discussion. We loved your suggestion, and this sparked a discussion within the time to explore what the learners think of the Global Values, and if they even find them useful. We started a discussion on X, Mastodon, and Discord as our channels to understand the voice of our community and ran a poll asking, **"Do you find the 'Global values' on the CSS pages useful?"** in each of these places. Here are the results:

**[X:  56 votes](https://x.com/MozDevNet/status/1831618522256175240)**
- Yes, good to see!, 78.6%
- No, 12.5%
- A link should be enough, 8.9%

**[Mastdon: 58 votes](https://mozilla.social/@MDN/113084106722251300)**
- Yes, good to see!, 76%
- No, 12%
- A link should be enough, 12%

**[Discord: 46 votes](https://discord.com/channels/1009925603572600863/1043638689802231809/1281177049993576513)**
- Yes, good to see!, 91%
- No, 2%
- A link should be enough, 7%

In total, we have collected **160 votes across 3 platforms** over the weekend, and received an overwhelming answer that _Yes, Global values are good to see._ This does not mean we won't ever change it, this only means that we will ask our community again in a few months and see if they are less useful to folks.

I would love to hear your thoughts, and how we can improve MDN! 🌟
